File: 02.PYTHON/D_Secuencia_Neg_Pos.py
print("")
print("DECISIONES ANIDADAS")
print("")

num = int(input("Escriba un número Entero:  "))

if(num > 0):
    print("El número es positivo, ")
elif(num < 0):
     num = num * (-1)
     print("El número es negativo, ")

if(num >=1 and num <=9):
    print("y tiene un dígito.")
elif(num >= 10 and num <= 99):
    print("y tiene 2 dígitos.")
elif(num >=100 and num <=999):
    print("y tiene 3 dígitos")
else:
    print("y tiene más de 3 dígitos")

      

File: 01.JAVA/D_Secuencia_Neg_Pos.java
import java.util.Scanner;
public class D_Secuencia_Neg_Pos{
    public static void main(String[] args){

        int num = 0;

        Scanner entrada = new Scanner(System.in);

        System.out.println("");
        System.out.println("DECISIONES ANIDADAS");
        System.out.println("");

        System.out.println("Escriba un número Entero:  ");
        num = entrada.nextInt();

        if(num > 0){
            if(num >= 10 && num <= 99){
                System.out.println("El número es positivo y tiene 2 dígitos.");
            }else{
                System.out.println("El número es positivo y NO tiene 2 dígitos.");
            }
        }else{
            if(num < 0){
                if(num >= -999 && num <= -100){
                    System.out.println("El número es negativo y tiene 3 dígitos.");
                }else{
                    System.out.println("El número es negativo y NO tiene 3 dígitos.");
                }

          }
        }

 }
}